packages/widget-v2/src/pages/SwapPage/SwapPage.tsx

```diff
@@ -111,27 +93,33 @@ export const SwapPage = () => {
         }));
         tokenAndChainSelectorModal.hide();
       },
+      selectedAsset: getClientAsset(sourceAsset?.denom, sourceAsset?.chainID),
```

Comment:
we don't want to pass selectedAsset for the asset selectors

Reply:
![ATOM](https://github.com/user-attachments/assets/7fe9da3c-dab4-4cfe-a277-2ab88ad1accd)
the selectedAsset is for rendering the asset in the search input but we only want that for the chain selector / chain selection step